requirements.in

```diff
@@ -0,0 +1,24 @@
+alembic
+arrow
+bleach
+cached-property
+docutils
+Flask==0.10.1
+Flask-Admin
+Flask-Login==0.2.11
```

Comment:
When upgrading Flask-Security, this should be upgraded as well. The Flask-Login upgrade requires code changes, so it's beyond the scope of this change.